test/e2e/utils/olm.go

```diff
@@ -259,6 +263,18 @@ func (k TestKubernetes) InstalledCSVEnv(envName string, sub *coreos.Subscription
 	return envVars[index].Value
 }
 
+func (k TestKubernetes) SetRelatedImagesEnvs(sub *coreos.Subscription) {
+	csv, err := k.InstalledCSV(sub)
+	ExpectNoError(err)
+
+	envVars := csv.Spec.InstallStrategy.StrategySpec.DeploymentSpecs[0].Spec.Template.Spec.Containers[0].Env
+	for _, envVar := range envVars {
+		if strings.HasPrefix(envVar.Name, "RELATED") {
+			os.Setenv(envVar.Name, envVar.Value)
+		}
+	}
```

Comment:
I'm confused why we need to set these Env vars on the node running the tests, what am I missing?

Reply:
The VersionManager will crash with downstream since they refer env. But those envs aren't available on host that runs the tests.